package.json

```diff
@@ -52,6 +53,8 @@
 	],
 	"license": "MIT",
 	"dependencies": {
+		"@typescript-eslint/eslint-plugin": "^8.0.0",
```

Comment:
Is upstream likely to introduce new rules/changed behaviours in 8.1.x etc.? If so, that'd be a breaking change for our CI, so these should be exactly pinned (like we do with eslint-plugin-jsdoc).

Reply:
It has the following in their section titled "A change to the plugins *shall not* be considered breaking if it:"

> Adds additional checks to an existing rule that causes new reports in a small-to-medium set of cases in an average codebase.

([source](https://typescript-eslint.io/users/versioning#eslint-plugin))

Let's pin this (and `@typescript-eslint/parser`, as I understand it, they need to be in sync) exactly.